stress/shell/src/main/java/alluxio/stress/cli/StressMasterBench.java

```diff
@@ -236,7 +266,7 @@ public MasterBenchTaskResult runLocal() throws Exception {
 
     List<Callable<Void>> callables = new ArrayList<>(mParameters.mThreads);
     for (int i = 0; i < mParameters.mThreads; i++) {
-      callables.add(new BenchThread(context, mCachedFs[i % mCachedFs.length]));
+      callables.add(getBenchThread(context, i));
```

Comment:
why we change the `mCachedFs` to `i`?

Reply:
This is moving the new to a function, and pass index to the function